src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -89,6 +122,16 @@ function CardSection() {
                 )}
                 {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
             </View>
+            {billingStatus?.isRetryAvailable !== undefined && (
```

Comment:
Can you confirm if there are any other checks required here? `hasPurchases` or not free trial, etc.?

Reply:
I think it's fine. BillingStatus is created after various checks based on card/billing data so it should be sufficient to only look for that.